cudax/include/cuda/experimental/__device/device.cuh

```diff
@@ -11,7 +11,13 @@
 #ifndef _CUDAX__DEVICE_DEVICE
 #define _CUDAX__DEVICE_DEVICE
 
-#include <cuda/std/detail/__config>
+#include <cuda_runtime_api.h>
+// cuda_runtime_api needs to come first
+
+#include <cuda/std/__cuda/api_wrapper.h>
+
+#include "cuda/std/__cccl/attributes.h"
+#include "cuda/std/__type_traits/decay.h"
 
```

Comment:
All those should come after the system header include guard and we are missing the mandatory `__config` include